test/integration/Ipv8Connector.spec.ts

```diff
@@ -69,26 +71,25 @@ describe('Ipv8Connector.ts', function () {
     expect(did).to.eq(peers.employee.did)
   })
 
-  it('should be able to verify an attested claim', function (done) {
+  it('should be able to verify an attested claim', async function () {
     this.slow(5000)
     this.timeout(10000)
     const brewerConnector = new Ipv8Connector()
     const employeeConnector = new Ipv8Connector()
     brewerConnector.configure(peers.brewer.url)
     employeeConnector.configure(peers.employee.url)
 
-    employeeConnector.observeVerificationRequests(peers.employee.did, { did: peers.brewer.did }).then(result => {
-      const subscription = result.observable.subscribe({
-        next: c => {
-          employeeConnector.ipv8AttestationClient.allowVerify('eGU/YRXWJB18VQf8UbOoIhW9+xM=', c.claim.data)
-          subscription.unsubscribe()
-        },
-        error: e => assert.fail('Error when observing:' + e.message)
-      })
+    const observeResult = await employeeConnector.observeVerificationRequests(peers.employee.did, { did: peers.brewer.did })
+    await observeResult.readPromise
```

Comment:
This says "readPromise", apparently awaiting `undefined` doesn't cause an error?

Reply:
Apparently not... (which make sense now I am thinking of it since it is a keyword an probably just stripped away if no promises are involved). After fixing this typo I realized that this `readyPromise` is actually not really useful.

The `observingPeer` is trying to find itself in it's own list of available peers. Since a peer cannot attest it's own claims it won't find itself as available peer either. I removed the `readyPromise` but kept the improved test for possible usage in the future